src/components/atoms/image/image.tsx

```diff
@@ -0,0 +1,13 @@
+import { Image as ChakraImageComponent } from "@chakra-ui/react";
+
+interface ImageProps extends React.ComponentProps<typeof ChakraImageComponent> {
+  //Add additional prop definitions here
+}
+
+const DefaultImage = (props: ImageProps) => {
+  return (
+    <ChakraImageComponent {...props} />
+  );
+};
+
+export default DefaultImage;
```

Comment:
```suggestion
```

Please delete this component as it's not needed. We are using next/image because we are leveraging Next.js. There should be some instances of next/image in the header component to help you learn how to leverage next/image.